job-dsls/jobs/srcclr_scan_seed.groovy

```diff
@@ -0,0 +1,88 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def repoList = [
+        'lienzo-core',
+        'drools'
+]
+
+for (repo in repoList) {
+
+    def jobName = 'srcclr-scan-' + "${repo}"
+
+    job(jobName) {
+
+        description('Job responsible for SourceClear verification of' + "${repo}")
```

Comment:
it can be replaced by `description("Job responsible for SourceClear verification of ${repo}")`

Reply:
the mixing ' and " is a little bit against my taste ;)